src/builtins/BuiltinTemporal.cpp

```diff
@@ -69,6 +69,13 @@ void checkTemporalPlainDateTime(ExecutionState& state, const Value& value)
     }
 }
 
+void checkTemporalDuration(ExecutionState& state, const Value& value)
+{
+    if (!(value.isObject() && value.asObject()->isTemporalDurationObject())) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "Invalid type");
```

Comment:
`checkTemporalDuration` is invoked only inside BuiltinTemporal.cpp file. So, please add `static` keyword to this function.
Or, instead of invoking this function, using a macro would be another option because this function body is composed of simple check codes.